### DISCUSSION TITLE
Tarea 1 LAB 03.01

### DISCUSSION THREAD

**Discussion by ThomasPineda** [rramosp/ai4eng.v1#55] [03 - GENERACION DE MODELOS]
Ya realice el código de la tarea 1, me da lo que tenia que dar según la verificación pero cuando se va a graficar la lineal de frontera aparece esto: 
![Captura de pantalla 1943-04-29 a la(s) 4 00 07 p  m](https://user-images.githubusercontent.com/86691796/126395161-0c4390b9-9ef5-45cd-9ea2-5611ee0d93d3.png)
![Captura de pantalla 1943-04-29 a la(s) 4 00 23 p  m](https://user-images.githubusercontent.com/86691796/126395167-d8666af4-4e40-4610-b4a2-a9c11ca5ee58.png)
 
y no se que pasa ni como solucionarlo

**Comment by Felipe-RA** [comment]
Saludos Thomas

Esta excepción de tipo `TypeError` ocurre cuando se tiene una operación incompatible. En este caso una **secuencia** multiplicada por un float.

En tu línea 78 tienes:

`p = (preds*1.).reshape((n,n))`

Verifica el tipo de dato de esta operación
